modules/RTC/RTC.js

```diff
@@ -93,9 +93,13 @@ export default class RTC extends Listenable {
         });
     }
 
-    onIncommingCall (event) {
+    /**
+     * Initializes the data channels of this instance.
+     * @param peerconnection the associated PeerConnection.
```

Comment:
Missing type description here. Is it TraceablePeerConnection or WebRTC PeerConnection ?